config/config.toml

```diff
@@ -1,151 +1,85 @@
-# An example config showing all of the toml variables used.
-
-# Copy to /etc/soci-snapshotter-grpc/config.toml
-# to use on your system.
-
-# NOTE: Many variables set to zero are just an indicator
-# to use the built-in default. These values may change over time,
-# which is why the config uses zeroes.
-# Please see docs/config.md for more info on what these variables do.
-
-# TODO: Can we put these inside dedicated TOML vars to refer to them?
-
-# config/fs.go FSConfig
-http_cache_type=""
-filesystem_cache_type=""
-resolve_result_entry=0
-debug=false
-disable_verification=false
-max_concurrency=0  # Actually zero
-no_prometheus=false
-mount_timeout_sec=0
-fuse_metrics_emit_wait_duration_sec=0
-
-## config/config.go Config
-
-metrics_address=""
-metrics_network="" # Uses default metrics network
-# no_prometheus=true # Defined above, can't be redeclared
-debug_address=""
-metadata_store="db"
-skip_check_snapshotter_supported=false
-
-[pull_modes.parallel_pull_unpack]
-enable=false
-max_concurrent_downloads=-1
-max_concurrent_downloads_per_image=3
-concurrent_download_chunk_size=""
-max_concurrent_unpacks=-1
-max_concurrent_unpacks_per_image=1
-discard_unpacked_layers=false
-
-[pull_modes.parallel_pull_unpack.decompress_streams."gzip"]
```

Comment:
Should this config still be printed?

Reply:
Probably not. By default these aren't initialized. With that said, we do also store default values for `parallel_pull_unpack` mode even when it's disabled so maybe not a bad idea to include this.

Reply:
Yeah I decided to just do it since we do have default values for `parallel_pull_unpack`, and even if the binary is missing on their system for some reason this won't instantiate unless they enable the pull mode.